Lib/test/datetimetester.py

```diff
@@ -413,6 +413,9 @@ def test_offset_boundaries(self):
                 with self.assertRaises(ValueError):
                     timezone(delta)
 
+    def test_comparison_with_tzinfo(self):
+        self.assertNotEqual(timezone.utc, tzinfo())
```

Comment:
I was going to say harumph about constructing a `tzinfo` object, but this is apparently done elsewhere in the tests, with this comment:

```
        # In order to allow subclasses to get pickled, the C implementation
        # wasn't able to get away with having __init__ raise
        # NotImplementedError
```

In any case, I think maybe a comment referencing `bpo-37915` would be a good idea.